flow/connectors/clickhouse/cdc.go

```diff
@@ -215,3 +215,20 @@ func (c *ClickhouseConnector) SyncFlowCleanup(ctx context.Context, jobName strin
 
 	return nil
 }
+
+func (c *ClickhouseConnector) RemoveTableEntriesFromRawTable(ctx context.Context, req *protos.RemoveTablesFromRawTableInput) error {
+	for _, tableName := range req.DestinationTableNames {
+		// Better to use lightweight deletes here as the main goal is to
+		// not have the rows in the table be visible by the NormalizeRecords'
+		// INSERT INTO SELECT queries
```

Comment:
wouldn't this only be true if the `SELECT` statement was using `FINAL`?

Reply:
Nope, I queried the raw table and checked. Basically according to the [docs](https://clickhouse.com/docs/en/guides/developer/lightweight-delete), internally in Clickhouse the row is soft deleted but all user queries to the table will not include the rows deleted like this